Abstract Dungeon (FR base)/sheet.json

```diff
@@ -4,116 +4,116 @@
       "descriptiontranslationkey": "attributeinfo"
     },
     {
       "attribute": "attrait",
-      "displayname": "CompÃ©tences 1 :",
-      "displaytranslationkey": "Skills 1 :",
+      "displayname": "Compétences 1 :",
+      "displaytranslationkey": "Skills 1:",
       "type": "text",
       "default": "Traits",
-      "description": "Change les compÃ©tences pour correspondre Ã  votre type de jeu",
+      "description": "Change les compétences pour correspondre à votre type de jeu",
       "descriptiontranslationkey": "skillinfo"
     },
     {
       "attribute": "atequipement",
-      "displayname": "CompÃ©tences 2 :",
-      "displaytranslationkey": "Skills 2 :",
+      "displayname": "Compétences 2 :",
+      "displaytranslationkey": "Skills 2:",
       "type": "text",
       "default": "Equipement",
-      "description": "Change les compÃ©tences pour correspondre Ã  votre type de jeu",
+      "description": "Change les compétences pour correspondre à votre type de jeu",
       "descriptiontranslationkey": "skillinfo"
     },
     {
       "attribute": "attresor",
-      "displayname": "CompÃ©tences 3 :",
-      "displaytranslationkey": "Skills 3 :",
+      "displayname": "Compétences 3 :",
+      "displaytranslationkey": "Skills 3:",
       "type": "text",
-      "default": "TrÃ©sors",
-      "description": "Change les compÃ©tences pour correspondre Ã  votre type de jeu",
+      "default": "Trésors",
+      "description": "Change les compétences pour correspondre à votre type de jeu",
       "descriptiontranslationkey": "skillinfo"
     },
     {
       "attribute": "desc1",
-      "displayname": "Description CaractÃ©ristique 1 :",
-      "displaytranslationkey": "attribute description1 :",
+      "displayname": "Description Caractéristique 1 :",
+      "displaytranslationkey": "attribute description1:",
       "type": "text",
       "default": "Trancher, Endurer, Briser, Sauter, Grimper, Plier, Nager, Souffler, Courir, Broyer, Pousser, Tirer, Soulever, Porter, Encaisser, Arracher, Lancer",
-      "description": "Change les descriptions pour correspondre Ã  votre type de jeu",
+      "description": "Change les descriptions pour correspondre à votre type de jeu",
       "descriptiontranslationkey": "descinfo"
     },
     {
       "attribute": "desc2",
-      "displayname": "Description CaractÃ©ristique 2 :",
-      "displaytranslationkey": "attribute description2 :",
+      "displayname": "Description Caractéristique 2 :",
+      "displaytranslationkey": "attribute description2:",
       "type": "text",
-      "default": "Esquiver, Parer, Equilibrer, Basculer, Rouler, Viser, Danser, Attraper, Ramasser, DÃ©sarmer, Nouer, S'Ã©chapper, DÃ©vier, Transpercer, Se retourner",
-      "description": "Change les descriptions pour correspondre Ã  votre type de jeu",
+      "default": "Esquiver, Parer, Equilibrer, Basculer, Rouler, Viser, Danser, Attraper, Ramasser, Désarmer, Nouer, S'échapper, Dévier, Transpercer, Se retourner",
```

Comment:
The apostrophe in this string is currently causing the string to be truncated in the campaignsettings page on Roll20

I think this is a bug on Roll20's side; I tried putting backslashes before the apostrophe, but that didn't fix anything

I don't know of a way to phrase this en français without an apostrophe, though, so I left it unchanged for now

Reply:
@wing-it could you weight in on this issue?